packages/wc-shoelace/elements/sh-sl-autocomplete.ts

```diff
@@ -20,7 +25,7 @@ export class ShSlAutocomplete extends LitElement {
   @property({ type: String })
     inputValue = ''
 
-  @property({ type: Boolean, reflect: true })
```

Comment:
That is reflect?

Reply:
Yes. Reflect is to keep a sync between HTML element attribute and property value.

https://lit.dev/docs/components/properties/#reflected-attributes